contracts/utils/GameOwner.sol

```diff
@@ -44,7 +46,7 @@ contract GameOwner is Ownable {
      * Modifier which restricts method execution to onlyGameOwner address
      */
     modifier onlyGameOwner() {
-        require(gameOwnerAddress == _msgSender(), "GameOwner: caller is not the game address");
+        require(isGameOwnerAddress(), "GameOwner: caller is not the game address");
```

Comment:
We already had this method and its being used on multiple places. Hence i figured i can just call it instead of having copy pasted line of code in the require.

This is just a styling and code optimization ticket.